Library/Formula/puddletag.rb

```diff
@@ -41,6 +41,12 @@ def install
     end
 
     cp_r buildpath/"source/.", buildpath
+    inreplace buildpath/"puddlestuff/puddleobjects.py" do |s|
+      s.gsub! /QPalette.Window/, "pal.window()"
```

Comment:
The puddletag version 1.0.5 (which is the actual tagged one in the upstream github repo) does only support an older version of pyqt (4.11.3). Since the pyqt formula got an update, this inreplace is necessary for puddletag in version 1.0.5 to work (for compatibility).

The master for puddletag in the github includes exactly this fix.

Once puddletag will get a new version tag, this line can hopefully be deleted.

If there is a method to install a specific version of pyqt via brew please tell me.